examples/bcic_iv_2a_cropped.py

```diff
@@ -24,7 +24,7 @@
 )
```

Comment:
can you call your examples plot_... so they get rendered in the doc. 

See here they are there https://164-232335424-gh.circle-artifacts.com/0/dev/auto_examples/index.html  but it's not executed by the CircleCI. thanks